Configure With Environment Variables Section Update README.md (#188)
aixigo/PREvant: api/README.md

## Configure With Environment Variables

> As stated above, PREvant utilizes [figment][1] to resolve configuration values from file, environment variables, and CLI options. The following examples provide a reference on how environment variables can be used to configure PREvant:

Reviewer: ```suggestion
As stated above, PREvant utilizes [figment][1] to resolve configuration values from file, environment variables, and CLI options. The following example shows how environment variables can be used to configure PREvant:
```
